### DISCUSSION TITLE
HomeScreen: Open a note by default

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#722] [Feature Requests]
A user has requested the ability to open to a note by default, as that way he can easily link to other notes. I imagine this would also be a nice way to create a custom dashboard.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by vHanda** [comment]
This would be nice for vimwiki compatibility as the convention is to open `index.md`

**Comment by gfarrell** [comment]
Also interested in the potential for vimwiki integration. Am currently looking at a way to migrate to vimwiki and am lacking a good ios app to help with that. Will happily pay for the app.
